client/site-profiler/utils/format-ms-value.ts

```diff
@@ -0,0 +1,18 @@
+import { I18N } from 'i18n-calypso';
+
+export const formatMsValue = (
+	msValue: number,
+	translate: I18N[ 'translate' ]
+): React.ReactElement | string | number => {
```

Comment:
Nit: Can't we import the `translate` function here and use it from the function? It seems a bit redundant to me to always pass it with each call.

```suggestion
import { translate } from 'i18n-calypso';

export const formatMsValue = (
	msValue: number,
): React.ReactElement | string | number => {
```

Reply:
Yep, this works really well, thanks for the suggestion

https://github.com/Automattic/wp-calypso/pull/91806/commits/232aca9b9213969f20704ae5d93f52ee89b9e336